### ISSUE TITLE
PPO1 declines after 530 iterations

### ISSUE THREAD

**Issue by ecada** [openai/baselines#710]
PPO1 run on 16 processes declines after 530 iterations with random seed 0:

<img width="219" alt="screen shot 2018-11-12 at 09 31 10" src="https://user-images.githubusercontent.com/18642218/48330485-bb171800-e65d-11e8-8425-fdcddd2477e6.png">

Also why is the clip_param multiplied by the learning rate in PPO1, the constant version has a much better learning curve in the mujoco domain? 

<img width="526" alt="screen shot 2018-11-12 at 14 23 34" src="https://user-images.githubusercontent.com/18642218/48344590-97b49300-e686-11e8-8bd5-cba36761148a.png">

**Comment by pzhokhov** [comment]
Hi @ecada ! Which mujoco environment are you using ? 
Clip parameter in ppo1 is not multiplied by learning rate, rather, by the annealing factor of the learning rate. I personally have not experimented enough with mujoco to know if the annealing of learning rate and clip factor is justified. @joschu @olegklimov any thoughts? 
As for decline of the rewards - that likely can be avoided by tuning hyperparameters (as your own experiments with constant vs linear schedule show)... Also, if you keep training, does the reward keep declining or does it rather level (at least approximately)?

**Comment by ecada** [comment]
Hi again @pzhokhov ! I'm using mujoco 1.50 and the default given hyper parameters for the Humanoid-v2 environment with random seed 0. In the default version the 'linear' learning rate is used which decays over time and that is fed into the learning rate placeholder that is multiplied by the clip parameter. It's shows the same pattern in 3e7 timesteps of training. I was curious why the default version is linear. 

![screen shot 2018-11-13 at 22 10 45](https://user-images.githubusercontent.com/18642218/48436999-447b3700-e791-11e8-81d5-a5a3732a2cf5.png)

**Comment by pzhokhov** [comment]
I see... I suspect linear annealing of the clip factor and learning rate is advantageous in some other mujoco environments, so we transferred it to humanoid without tuning specifically on humanoid (the hyperparameters in run_mujoco.py are uncannily similar to ones in run_humanoid.py). Again, @joschu @olegklimov please feel free to override this if I am wrong on that.

**Comment by joschu** [comment]
Yeah the linear schedule helped a little bit on the 2D tasks. I'm not sure if it helps on the humanoid, but it seems sensible--in general stepsize annealing helps.

**Comment by ecada** [comment]
Thank you @pzhokhov and @joschu for clearing it up. I'm grateful for all the valuable feedback and the way this library is maintained. 🥇
